### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/wpf-winforms/01-login.md [hunk 1]
```diff
@@ -78,6 +78,6 @@ The `Auth0User` has the following properties:
 
 ## Download the samples
 
-Browse the samples on GitHub from [here](https://github.com/auth0/Auth0.WinformsWPF/tree/master/samples).
```
Comment (mpaktiti) [inline articles/native-platforms/wpf-winforms/01-login.md:81]: @mehreencs87 Please remove this paragraph and instead use the seed template for this. 
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects
Thanks.

**Comment by mpaktiti** [comment]
Pending sample review and merge: https://github.com/auth0-samples/auth0-winformsWPF-samples/pull/1

**Comment by mehreencs87** [comment]
@mpaktiti updated, check the second commit,

**Comment by mpaktiti** [comment]
We need to merge the [sample PR](https://github.com/auth0-samples/auth0-winformsWPF-samples/pull/1) before this one.

**Comment by mpaktiti** [comment]
Hi @mehreencs87 can you rebase to resolve the conflicts? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti resolved here https://github.com/auth0/docs/pull/2069

**Pull request closed without merging**